lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -302,8 +303,7 @@ def create(self, trans, payload, **kwd):
             if not os.path.exists(installed_repository_file):
                 raise exceptions.MessageException("Repository file '%s' not found.")
```

Comment:
This got lost in the rebase:
```suggestion
                raise exceptions.RequestParameterInvalidException("Workflow file '%s' not found" % installed_repository_file)
```